lib/src/includes/signed_video_sign.h

```diff
@@ -262,11 +261,11 @@ signed_video_get_nalu_to_prepend(signed_video_t *self,
  *   } else {
  *     size_t sei_size = 0;
  *     status = signed_video_get_sei(sv, NULL, &sei_size);
- *     // The first call of the function is for getiing the sei_size.
- *     // The Second call is to  get the sei.
- *     uint8_t *sei = malloc(sei_size);
- *     status = signed_video_get_sei(sv, sei, &sei_size);
+ *     // The first call of the function is for geting the |sei_size|.
+ *     // The second call is to  get the sei.
  *     while (status == SV_OK && sei_size != 0) {
+ *       uint8_t *sei = malloc(sei_size);
+ *       status = signed_video_get_sei(sv, sei, &sei_size);
  *         break;
```

Comment:
This is incorrect. It will not compile if used and it dose not match the principle used in our tests for example

Reply:
*     while (status == SV_OK && sei_size != 0) {
 *       uint8_t *sei = malloc(sei_size);
 *       status = signed_video_get_sei(sv, sei, &sei_size);
 *       if (status != SV_OK) {
 *          // True error. Handle it properly.
 *          }
 *       status = signed_video_get_sei(sv, NULL, &sei_size); 
 *       }
 For it to work I need to update  the sei_size  every iteration to  allocate  the write size for the next sei .
 Am I missing something?

Reply:
The code is there in `pull_nalus()` in the test helper function